doc/source/api/metadata.rst

```diff
@@ -10,20 +10,117 @@ Goals and Scope
 
 The metadata API provides information on the primary data objects
 available via the GA4GH API, and facilities to organize primary data
-objects.
+objects. Main goals of the metadata definitions are to provide
 
-The current metadata API is immature and will evolve in future.
+* standardized use of common attributes/values
+* a schema of how (core) objects relate to each other
 
```

Comment:
Removed. Basically referred to the concept of main objects in the hierarchy
	Individual <=> BioSample <=> (VariantCall or such)

… vs. additional information objects (Experiment, Ontologies …).

> On 30 Nov 2015, at 18:29, Mark Diekhans <notifications@github.com> wrote:
> 
> what is a `core' object? Throwing in a new term is confusing.
> 
>